mappings/net/minecraft/world/gen/WorldProvider.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS none/auf net/minecraft/world/gen/WorldProvider
 	FIELD b world Lnone/aiw;
 	FIELD c biomeProvider Lnone/ajn;
+	FIELD d waterEvaporates Z
+	FIELD e noSky Z
```

Comment:
I named it this way because this variable's main use is in sky light calculations. The only other thing that it does is make the world height half as tall as normal. `isVoid` is a good name but I think it loses some important information.